src/core/internal/spinlock.d

```diff
@@ -52,7 +52,7 @@ shared struct SpinLock
     {
         import core.time;
         if (k < pauseThresh)
-            return pause();
+            return core.atomic.pause();
```

Comment:
Correct if im wrong but this is more than a simple refactoring as the new inline assembly differs:
Old:
```
rep
nop
```
New:
```
naked
rep
nop
ret
```